### ISSUE TITLE
Valid 'su' command found error

### ISSUE THREAD

**Issue by MikeRalphson** [xxsl/DebianKit#1]
Not expecting this to be fixed here (now), but just in case anyone else happens upon this.

To get the `Valid `su` command found` message to pass, the `su` command on your Android device must be patched/replaced and have permissions beginning `-rws` in **every** directory where it exists on your `$PATH`. On my device this was `/system/sbin`, `/system/bin` and `/system/xbin`.

**Comment by xxsl** [comment]
I don't know if you mean the Android APP instead of the script, 

There is a dirty check here in
https://github.com/xxsl/DebianKit/blob/f9d97738f250422e6198a3923ffc1fac1f0f4d32/app/src/org/dyndns/sven_ola/debian_kit/DebianKitActivity.java#L263

to make sure su with mode 6755. I think this way could be improved later if there's a plan to maintain this sorry that.

**Comment by MikeRalphson** [comment]
Yes, that's the check, I think there is a missing `break` statement.
